packages/public-docsite-setup/src/constants.ts

```diff
@@ -1,4 +1,4 @@
-import { ManifestVariant } from './types';
+import type { ManifestVariant } from './types';
```

Comment:
This package shouldn't use `export type` because it's consumed by older branches (ideally this would be enforced by tooling in some way, at least lint rules, but I couldn't think of an easy way to do that)